tools/bin/bu_tag_push.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
This looks like custom bu logic. We likely shouldn't be using a fork and instead using a clone.